lib/Dialect/Linalg/Transforms/LowerToLLVMDialect.cpp

```diff
@@ -594,10 +594,7 @@ class TransposeOpConversion : public LLVMOpLowering {
     auto tranposeOp = cast<TransposeOp>(op);
     BaseViewConversionHelper helper(op, tranposeOp.getViewType(), rewriter,
                                     lowering);
-    IndexType indexType = helper.indexType;
-    ViewType viewType = helper.viewType;
-    LLVMType elementTy = helper.elementTy, int64Ty = helper.int64Ty,
-             viewDescriptorPtrTy = helper.viewDescriptorPtrTy;
```

Comment:
Seems like some of these (like `indexType`) can be removed as a member of `BaseViewConversionHelper` as well?

Reply:
Yes, there are some attributes that no other part of the code is using. I will remove it and update the pull request.

Reply:
Thanks